bin/cask

```diff
@@ -236,26 +236,48 @@ def find_best_emacs():
     else:
         return 'emacs'
 
+def inside_emacs_24():
+    """Whether Cask is running inside Emacs-24.
+
+Emacs 24 is rather inconsistent in its use of the ``$EMACS`` and
+``$INSIDE_EMACS`` environment variables, so we try here to test if Cask is
+being run inside Emacs (either in a shell or with ``M-x compile``).
+
+Return ``True`` when running in Emacs 24, ``False`` otherwise.
```

Comment:
Yes, like the other docstrings.  Emacs' built-in Python Mode automatically does that when you press `M-q` in a docstring.